### PR TITLE
Add auton action shutdown

### PR DESCRIPTION
**Pull request by team6101**
# Description

We added the functions so our auton actions actually stop.

# Checklist before marking this PR as ready for review

-   [x] All the PR checks are passing (you see the green checkmark)
-   [x] I have written comments on the PR where reviewers may have questions

### CODE REVIEW

File: src/main/java/frc/robot/Vision/AutonRotateAction.java
Comment (team6101) [inline src/main/java/frc/robot/Vision/AutonRotateAction.java:1]: this wasn't actually doing anything useful, so we deleted it

File: src/main/java/frc/robot/auton/AutonIntake.java [hunk 1]
```diff
@@ -10,4 +10,10 @@ public boolean isDone() {
 
     @Override
     public void initiate() {}
+
+    @Override
+    public void shutdown() {
+        // TODO Auto-generated method stub
+        throw new UnsupportedOperationException("Unimplemented method 'shutDown'");
+    }
```
Comment (team6101) [inline src/main/java/frc/robot/auton/AutonIntake.java:18]: AutonIntake isn't set up properly right now, so we don't have anything to shutdown

File: src/main/java/frc/robot/Vision/AutonRotateAction.java
Comment (team6101) [reply src/main/java/frc/robot/Vision/AutonRotateAction.java:1]: correction, AutonRotate wasn't doing anything, we made AutonRotate do things and deleted this one

File: src/main/java/frc/robot/auton/AutonRoutes.java [hunk 2]
```diff
@@ -7,17 +7,6 @@
...
-        AutonAction
-    >(
-        List.of(
-            new AutonParallelAction(
-                new AutonSequentialAction(new AutonWait(1.5), new AutonIntake()),
-                new AutonMoveInches(-36)
-            )
-        )
-    );
-
```
Comment (team6101) [inline src/main/java/frc/robot/auton/AutonRoutes.java:20]: this isn't used and doesn't do what it says it does

File: src/main/java/frc/robot/auton/AutonShoot.java [hunk 3]
```diff
@@ -13,11 +12,9 @@ public class AutonShoot extends AutonAction {
     public boolean isDone() {
         if (Timer.getFPGATimestamp() >= revFinishedTime) {
             Robot.motors.getFeeder().set(0.2);
+            Robot.motors.getIntake().set(.2);
         }
         if (Timer.getFPGATimestamp() >= revFinishedTime + 1.0) {
-            Robot.motors.getFeeder().set(0);
-            Robot.motors.getLeftFlywheel().set(0);
-            Robot.motors.getRightFlywheel().set(0);
```
Comment (team6101) [inline src/main/java/frc/robot/auton/AutonShoot.java:20]: we call shutdown when we stop so we dont need to do this now

File: src/main/java/frc/robot/auton/ExtendArms.java [hunk 4]
```diff
@@ -11,8 +11,6 @@ public class ExtendArms extends AutonAction {
     @Override
     public boolean isDone() {
         if (Timer.getFPGATimestamp() >= extensionTime) {
-            Robot.motors.getLeftClimb().set(0);
-            Robot.motors.getRightClimb().set(0);
```
Comment (team6101) [inline src/main/java/frc/robot/auton/ExtendArms.java:15]: happens in shutdown

**Review by team6101 — COMMENTED** [review]
our code is now correctly spelled

File: src/main/java/frc/robot/InputtedCoDriverControls.java [hunk 5]
```diff
@@ -23,6 +23,9 @@ public static void onEveryFrame() {
             shootIntoSpeaker();
         }
         doShooterIntake();
+        if (controller.getYButton()) {
+            doFloorIntake();
```
Comment (Nanosplitter) [inline src/main/java/frc/robot/InputtedCoDriverControls.java:27]: Does this turn off if the button stops being pressed?

**Review by Nanosplitter — APPROVED** [review]
I'm good with this! Take a look at my comment though

**Pull request merged**